DEVELOPMENT.md

```diff
@@ -89,3 +59,36 @@ If you just want to download the sources without uploading your keys get the sou
 ```bash
 $ git clone https://github.com/developer-portal/content.git
 ```
+
+
+### Using a Docker container
+
+If you don't have Docker installed, you can check Fedora Developer Portal on[installing Docker with libvirt provider](https://developer.fedoraproject.org/tools/docker/about.html).
+
+Docker container provides a simple way how to run the development instance of Developer Portal. Following command will start the Jekyll server in a container (with very similar output):
+
+```
+$ sudo docker run -it --rm developerportal/devel
+Configuration file: /website/_config.yml
+            Source: /website
+       Destination: /website/_site
+      Generating...
+                    done.
+ Auto-regeneration: enabled for '/website'
+Configuration file: /website/_config.yml
+    Server address: http://172.17.1.16:8080/
+  Server running... press ctrl-c to stop.
+
+```
+
+The above command will serve the latest content available in Github repository. If you want to do some changes in a `website` repository and view them, you need to add argument `-v /path/to/your/repo:/opt/developerportal/website`:
+
+```
+$ sudo docker run -it --rm -v $PWD:/opt/developerportal/website:z developerportal/devel
+```
+
+In case you want to modify the `content` repository, you need to add argument `-v /path/to/content/repo:/opt/developerportal/website/content`:
+
+```
+$ sudo docker run -it --rm -v $PWD/content:/opt/developerportal/website/content developerportal/devel
```

Comment:
`:z` seems to be missing here to be able to generate the site.
Or does it work without it?